Installing ember-cli-deploy on ember-cli@1.13.0 fails

Steps to reproduce:

1. ember new my-app
2. ember install ember-cli-deploy
3. Boom.

```
Cannot find module 'ember-cli/lib/errors/silent'
Error: Cannot find module 'ember-cli/lib/errors/silent'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/Users/user/my-app/node_modules/ember-cli-deploy/lib/utilities/configuration-reader.js:4:19)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
```

--- a/lib/errors/silent.js
+++ b/lib/errors/silent.js
@@ -1,11 +1,12 @@
 'use strict';
 
-var silentError = require('silent-error');
+var SilentError = require('silent-error');
 var deprecate   = require('../utilities/deprecate');
 
-module.exports = function(){
-  deprecate('`ember-cli/lib/errors/silent.js` is deprecated,'+
-    ' use `silent-error` instead.', true);
-    
-  return silentError;
-};
+Object.defineProperty(module, 'exports', {
+  get: function () {
+    deprecate('`ember-cli/lib/errors/silent.js` is deprecated,'+
+      ' use `silent-error` instead.', true);
+    return SilentError; 
+  }
+});

--- a/lib/errors/silent.js
+++ b/lib/errors/silent.js
@@ -1,0 +1,11 @@
+'use strict';
+
+var silentError = require('silent-error');
+var deprecate   = require('../utilities/deprecate');
+
+module.exports = function(){
+  deprecate('`ember-cli/lib/errors/silent.js` is deprecated,'+
+    ' use `silent-error` instead.', true);
+    
+  return silentError;
+};

--- a/tests/unit/errors/silent-test.js
+++ b/tests/unit/errors/silent-test.js
@@ -1,0 +1,11 @@
+'use strict';
+
+var SilentError    = require('silent-error');
+var SilentErrorLib = require('../../../lib/errors/silent');
+var expect         = require('chai').expect;
+
+describe('SilentError', function() {
+  it('return silent-error and print a deprecation', function() {
+    expect(SilentErrorLib, 'returns silent-error').to.equal(SilentError);
+  });
+});

